### DISCUSSION TITLE
Got stuck on Step 4

### DISCUSSION THREAD

**Discussion by CodaCodr** [skills/.github#468] [GitHub Pages]
Hi

I'm working through the course to create a github blog. It worked fine until the end of step 4. I've waited ~5mins or more, refreshing does nothing.

How do I proceed?

**Comment by 1dropaflame** [comment]
same here, I think it's broken 👎 
I created issue here https://github.com/skills/.github/issues/483

**Comment by YunusOganPXL** [comment]
same problem here can anyone help?

**Comment by AnanyaP-WDW** [comment]
Yes there seems to be an issue with github actions.

**Comment by TheGreatRandino** [comment]
well it November, 10 months from the first post and I've got the same problem.
anybody know a workaround

**Comment by CodaCodr** [comment]
OP, here. It's clear to me this code is no longer maintained.

**Reply by TheGreatRandino** [reply]
So, would you suggest I NOT continue any more of these GitHub Skills?

**Reply by 1dropaflame** [reply]
I had opened an issue here https://github.com/skills/github-pages/issues/94
If you are still facing the issue then please reopen the ticket.

**Comment by CodaCodr** [comment]
That's not for me to say.

**Comment by msaltenna** [comment]
yup stuck on step 4

**Comment by petomer-codes** [comment]
hi, it is still not working, but you can get to step 5 by going to main branch -> github folder -> steps.

**Comment by ca1410** [comment]
Like file
